index.html

```diff
@@ -626,7 +626,7 @@ <h3>Information Model</h3>
             </section>
 
             <figure id="directory-class-diagram">
-                <img src="images/directory-class-diagram.svg"
+                <img src="images/registration_info-diagram.svg"
```

Comment:
The naming convention is dash separated names. See others: https://github.com/w3c/wot-discovery/tree/main/images

Reply:
Figure ID does not match the changed figure. Need to also change the citations.

Reply:
RegistrationInfo (in relation) does not match RegistrationInformation (class name).

Reply:
you are right, I change the names of the files, corrected the diagram, and updated the file in the index. Regarding the citations, I updated the id to enriched-td-class-diagram